.gitlab/custom-jobs-and-variables.yml

```diff
@@ -101,8 +74,10 @@ variables:
 .custom_job:
   artifacts:
     reports:
-      junit: junit.xml
```

Comment:
Not sure whether GitLab will be happy with a empty reports section. We may need to remove more than that.

Reply:
I see no complaints in gitlab logs. I removed the line because gitlab was generating warnings about a missing file or something like that.